README.md

```diff
@@ -221,7 +221,7 @@ The following options are valid inside a part configuration:
   Explicit list of all values that will be iterated when bumping that specific
   part.
 
-  Example::
+  Example:
```

Comment:
I think this was supposed to be a [restructured text code block](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#inserting-code-and-literal-blocks). It doesn't appear to be working though. I think that's because of indenting of the rest of the block. I wonder if we should use markdown code blocks so we also get syntax highlighting using tripple backticks and ini:
```ini
[bumpversion:part:release_name]
values =
  witty-warthog
  ridiculous-rat
  marvelous-mantis
```

Reply:
That does look much more readable indeed!

Reply:
I can edit all `ini` code blocks of the README with the triple backtick syntax, in this PR or a new one, if you think this is the way to go.

Reply:
I think in this PR is fine. We can update the title to fix typos + code blocks syntax. It's no real content change anyway which I think is the important part.